api/mesh/v1alpha1/fault_injection.proto

```diff
@@ -8,32 +8,52 @@ import "google/protobuf/duration.proto";
 import "mesh/v1alpha1/selector.proto";
 import "google/protobuf/wrappers.proto";
 
+// FaultInjection defines configuration of faults between dataplanes.
 message FaultInjection {
 
+  // List of selectors to match dataplanes that are sources of traffic.
   repeated Selector sources = 1;
 
+  // List of selectors to match services that are destinations of traffic.
   repeated Selector destinations = 2;
 
+  // Configuration defines several types of faults, at least one delay should be
+  // specified
   message Conf {
 
+    // Defines configuration of the delay
     message Delay {
+      // Percentage of requests on which delay will be injected
       google.protobuf.DoubleValue percentage = 1;
+      // Duration, the value of delay
```

Comment:
`The duration for which the response will be delayed`?
or something similar, `value of delay` could be more descriptive.